index.html

```diff
@@ -0,0 +1,265 @@
+                            required. has been the
+                            industry's standard</p>
+                        <li class="listbutton2">
+                            <ul><button type="button" class="button6"> html</button></ul>
+                            <ul><button type="button" class="button7"> bootstrap</button></ul>
+                            <ul><button type="button" class="button8"> Ruby</button></ul>
+                        </li>
+                    </div>
+                </div>
+                <button type="button" class="button9"> See Project</button>
+            </div>
+            <div class="card">
+                <!-- second card -->
+                <div class="listproject">
+                    <h2 class="art">Profesional Art Printing Data</h2>
+                    <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                        required. has been the
+                        industry's standard</p>
+                    <li class="listbutton2">
+                        <ul><button type="button" class="button6"> html</button></ul>
+                        <ul><button type="button" class="button7"> bootstrap</button></ul>
+                        <ul><button type="button" class="button8"> Ruby</button></ul>
+                    </li>
+                </div>
+            </div>
+            <button type="button" class="button9"> See Project</button>
+            </div>
+            <!-- third card -->
+            <div class="listproject">
+                <h2 class="art">Profesional Art Printing Data</h2>
+                <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                    required. has been the
+                    industry's standard</p>
+                <li class="listbutton2">
+                    <ul><button type="button" class="button6"> html</button></ul>
+                    <ul><button type="button" class="button7"> bootstrap</button></ul>
+                    <ul><button type="button" class="button8"> Ruby</button></ul>
+                </li>
+            </div>
+            </div>
+            <button type="button" class="button9"> See Project</button>
+            </div>
+            <!-- fourth card -->
+            <div class="listproject">
+                <h2 class="art">Profesional Art Printing Data</h2>
+                <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                    required. has been the
+                    industry's standard</p>
+                <li class="listbutton2">
+                    <ul><button type="button" class="button6"> html</button></ul>
+                    <ul><button type="button" class="button7"> bootstrap</button></ul>
+                    <ul><button type="button" class="button8"> Ruby</button></ul>
+                </li>
```

Comment:
- [ ] Kindly make sure you use semantic tag following the Web3C standard.
`It's supposed to be:`
`<ul>`
    `<li>JavaScript</li>`
     `<li>Ruby</li>`
`</ul>`
`And not the other way around. Please consider changing that.`